api/base/serializers.py

```diff
@@ -1539,3 +1539,15 @@ def create(self, validated_data):
             collection.add_pointer(node, auth)
 
         return self.make_instance_obj(collection)
+
+
+class MaintenanceStateSerializer(ser.ModelSerializer):
+
+    level = ser.SerializerMethodField()
+
+    def get_level(self, obj):
+        return obj.get_level_display()
```

Comment:
Might make sense to just serialize as an integer; that way the client can easily compare against it and decide what they want to do with it. Our API docs should make the semantics clear:

```
1 = Informational. Used most commonly for scheduled maintenance.
2 = Used for service outages
```